src/js/app.js

```diff
@@ -1,5 +1,48 @@
 import $ from "jquery";
+import slick from "slick-carousel";
 
 $(function() {
-  console.log('環境構築完了');
-})
+  $('.l-header__hamburger').click(function() {
+    $('.l-header__hamburger').toggleClass('active');
+    $('.l-header__sp-nav').toggleClass('active');
+
+    if ($("body").css("overflow") === "hidden") {
```

Comment:
シングルクォーテーションで揃えるようにしましょう
全体的に見渡してもらって修正をお願いします

```suggestion
    if ($('body').css('overflow') === 'hidden') {
```

Reply:
こちらもチェック漏れておりましたので修正しました。
お手数おかけしました・・・。